agimus_controller/agimus_controller/ocp_base.py

```diff
@@ -27,7 +30,9 @@ def dt() -> int:
 
     @abstractmethod
     def solve(
-        self, x0: np.ndarray, x_init: list[np.ndarray], u_init: list[np.ndarray]
+        self,
+        x_init: List[npt.NDArray[np.float64]],
+        u_init: List[npt.NDArray[np.float64]],
```

Comment:
```suggestion
        x_init: list[npt.NDArray[np.float64]],
        u_init: list[npt.NDArray[np.float64]],
```
Based on older @petrikvladimir comment we should not include typing